packages/eas-cli/src/log.ts

```diff
@@ -45,6 +45,10 @@ export default class Log {
     Log.warn(`› ${chalk.bold('--' + flag)} flag is deprecated. ${message}`);
   }
 
+  public static failed(message: string): void {
```

Comment:
```suggestion
  public static fail(message: string): void {
```
(to match tense of `succeed`. I'd also be open to renaming `succeed` to `success` and make this one `failure`)